features/assets.feature

```diff
@@ -15,18 +29,38 @@ Feature: Assets
     When I request "/assets/513a0efbed915d425e000002"
     And I should see "120613_Albania_Travel_Advice_WEB_Ed2_jpeg.jpg"
 
+  # TODO: REMOVE as this test is a duplicate.
+  #
+  # Duplicate of "Check assets with an xls extension are served correctly".
```

Comment:
There's an argument that this isn't a duplicate - are we confident that anything affecting asset type X will always affect asset type Y?

Reply:
Yes, there's nothing special about serving image files vs other files.

Asset Manager does have [a bit of special code for image files that don't exist](https://github.com/alphagov/asset-manager/blob/e8b25f21a72948117c7dcb084491af26b949805a/app/controllers/whitehall_media_controller.rb#L27) (unsurprisingly, it's to accommodate Whitehall), but that's not relevant for the test here, where we're testing files that do exist.